de/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+  <title>Frag nicht zum Fragen, frag einfach</title>
+  <meta property="og:title" content="Frag nicht zum Fragen, frag einfach" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="de_DE" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Frag nicht zum Fragen, frag einfach</h1>
+    <p>
+      Seitdem ich im Internet unterwegs bin, gibt es immer irgendwelche
+      Leute, die sowas schreiben:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Irgendein Java-Experte hier?
+      </p>
+    </blockquote>
+    <p>
+      Das ist aus mehreren Gründen eine schlechte Form. Was die Person 
+      <em>eigentlcich</em> fragt ist:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Gibt es hier irgendeinen Java-Experten, der mir bei meinem Problem, was
+        auch immer es ist, selbst wenn es nichts mit Java zu tun, helfen könnte -
+        oder könnte mir wenigstens irgendjemand antworten?
+      </p>
+    </blockquote>
+    <p>
+      Es gibt sehr viele Gründe weshalb Menschen, die wirklich das Wissen haben,
+      es nicht zugeben werden. Wenn du fragst, um etwas zu fragen, fragst du für
+      mehr als du denkst.
+    </p>
+    <p>
+      Du fragst Menschen die Verantwortung für das Lösen deines Problems zu
+      übernehmen. Du forderst das Selbstvertrauen in die Fähigkeiten der Personen
+      heraus. Unnötigerweise schließt du andere Leute vom Beantworten deiner Fragen
+      aus. Relativ oft beantworte ich Fragen zu anderen Programmiersprachen oder
+      Libraries die ich noch nie benutzt habe, weil die Antworten (aus der Sicht
+      eines Programmierers) Allgemeinwissen sind.
+    </p>
+    <p>
+      Alternativ sieht dies so aus...
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Ich habe eine Frage zu Java aber zum Formulieren der Frage bin ich zu faul,
+        außer es gibt hier jemand, der eventuell bereit wäre zu antworten.
+      </p>
+    </blockquote>
+    <p>
+      ...was einfach nur faul aussieht. Wenn du nicht gewillt bist deinen Beitrag
+      zum Beantworten deiner Frage beizutragen, warum sollten wir das?
+    </p>
+    <p>
+      Die Lösung ist nicht zum Fragen zu fragen, sondern einfach zu fragen. Es ist

```

Comment:
```suggestion
      Die Lösung ist nicht über das Fragen zu fragen, sondern einfach zu fragen. Es ist
```

Reply:
I know that "zum Fragen zu fragen" sounds really bad but I believe "über das Fragen zu fragen" is grammatically wrong.

Reply:
Maybe your right, I leave it up to your taste :3